app/controllers/admin/services_controller.rb

```diff
@@ -2,13 +2,24 @@ module Admin
 
   class ServicesController < ApplicationController
 
-    skip_before_action :verify_authenticity_token, only: :cancel_reservations_for_offline_instruments
+    skip_before_action :verify_authenticity_token
 
     def cancel_reservations_for_offline_instruments
       InstrumentOfflineReservationCanceler.new.cancel!
       render nothing: true
     end
 
+    def process_five_minute_tasks
+      self.class.five_minute_tasks.each do |worker|
+        worker.new.perform
+      end
+      render nothing: true
+    end
+
+    def self.five_minute_tasks
+      @five_minute_tasks ||= [AutoExpireReservation, EndReservationOnly, AutoLogout]
```

Comment:
Right, but if we don't `rescue` at all, then any exception will hit the `.transaction`, trigger the rollback, and then get re-raised.